pkgs/servers/openvscode-server/remove-node-download.patch

```diff
@@ -1,27 +1,21 @@
---- ./build/gulpfile.reh.js
-+++ ./build/gulpfile.reh.js
-@@ -277,8 +277,6 @@
+diff --git a/build/gulpfile.reh.js b/build/gulpfile.reh.js
+index a44941a1e73..946d3478004 100644
+--- a/build/gulpfile.reh.js
++++ b/build/gulpfile.reh.js
+@@ -265,8 +265,6 @@ function packageTask(type, platform, arch, sourceFolderName, destinationFolderNa
  			.pipe(util.stripSourceMappingURL())
  			.pipe(jsFilter.restore);
  
--		const nodePath = `.build/node/v${nodeVersion}/${platform}-${platform === 'darwin' ? 'x64' : arch}`;
+-		const nodePath = `.build/node/v${nodeVersion}/${platform}-${arch}`;
 -		const node = gulp.src(`${nodePath}/**`, { base: nodePath, dot: true });
  
  		let web = [];
  		if (type === 'reh-web') {
-@@ -296,7 +294,6 @@
+@@ -284,7 +282,6 @@ function packageTask(type, platform, arch, sourceFolderName, destinationFolderNa
  			license,
  			sources,
  			deps,
 -			node,
  			...web
  		);
  
-@@ -376,7 +373,6 @@
- 			const destinationFolderName = `vscode-${type}${dashed(platform)}${dashed(arch)}`;
- 
- 			const serverTaskCI = task.define(`vscode-${type}${dashed(platform)}${dashed(arch)}${dashed(minified)}-ci`, task.series(
--				gulp.task(`node-${platform}-${platform === 'darwin' ? 'x64' : arch}`),
```

Comment:
I'm pretty sure this line needs to be added back in to the patchfile and updated to remove the equivalent line in the newer version of `./build/gulpfile.reh.js`. It looks like the build is failing when trying to download Node.js: https://github.com/NixOS/nixpkgs/pull/167166/checks?check_run_id=5839872793